src/main/java/com/hpe/application/automation/tools/results/RunResultRecorder.java

```diff
@@ -1323,7 +1323,7 @@ public DescriptorImpl() {
         @Override
         public String getDisplayName() {
 
-            return "Publish Micro Focus tests result";
+            return "Publish tests result";
         }
```

Comment:
Please don't do this - since it will cause recognizing issues for our customers... they have additional steps and it might confuse..

Reply:
Please return this to micro focus since this is not only alm but also LR and uft..